env0/resource_environment_test.go

```diff
@@ -949,8 +950,7 @@ func TestUnitEnvironmentResource(t *testing.T) {
 	testApiFailures()
 }
 
-/* TODO
```

Comment:
uncommented the test and made some changes.
Will update the test in the next PR to test update as well.